templates/src/middleware/auth/index.js

```diff
@@ -0,0 +1,26 @@
+const authMiddleware = (req, res, next) => {
+  /** Expecting oathkeeper to pass on user identity in headers
+   *  (note nodejs converts all headers to lowercase)
+   *  1. X-User-Id
+   *  2. X-User-Email
+   * */
+  const hasKratosCookie = req.headers.cookie.match(/ory_kratos_session/);
```

Comment:
I don't think we should make the service have any knowledge of the kratos cookie. The only thing we would need it for is passing to kratos if we needed to call something like `/sessions/whoami` on behalf of the user, and we shouldn't need to do that. Having the service understand the cookie breaks the abstraction a bit.